### DISCUSSION TITLE
Welcome to Zapdos Discussions!

### DISCUSSION THREAD

**Discussion by cticenhour** [shannon-lab/zapdos#76] [News / Announcements]
## 👋 Welcome!
We’re exploring  the use of Discussions as a place to connect with other members of the Zapdos community, in order to align with MOOSE development and user group practices. We hope that you:
  * Ask questions.
  * Share ideas.
  * Engage with other Zapdos community members.

  To get started, feel free to comment below with an introduction of yourself and tell us about what you do with this community.

### A couple more notes...

* Issues and Pull Requests can be linked in discussions just like you're used to! Simply add the number for the issue in `#XXXX` format. Issues that would make for better Discussions can also be converted by maintainers -- look for a link on the right hand sidebar of the Issue. 

* The Google Group will remain as an archive, but please direct all new questions here!

**Comment by cticenhour** [comment]
I'll get the ball rolling with my own intro. I am Casey, a PhD student in Nuclear Engineering at NC State University. I have been working with MOOSE and Zapdos for the last 4-5 years, now as a member of the MOOSE development team. My research is focused on electromagnetics and low temperature plasma physics modeling and simulation (with some fusion work here and there). On the MOOSE team, I do some framework development and Mac platform support. 

By turning on Discussions, I hope to create a tighter connection between our community/user support and the code repository (and all the content and discussions in Issues and PRs!). This seems to have been a success for MOOSE so far, so I'm hoping to duplicate that here.

Tagging @csdechant, @keniley1, @skatersaurus, and @lindsayad

**Reply by cticenhour** [reply]
I would also like some feedback on possible Categories we could make to organize questions and posts. Feel free to throw that in here or in your own intro.

**Comment by keniley1** [comment]
I am Shane, and I am a PhD candidate at the University of Illinois at Urbana-Champaign. I mainly use Zapdos for studying plasma discharges over liquid water and non-equilibrium plasma chemistry. I am also the developer of [Crane](https://github.com/lcpp-org/crane), which is currently coupled to Zapdos to add all of the source and sink terms that occur in a plasma discharge due to chemical reactions. 

I haven't given categories too much thought, but I guess we could start with something similar to MOOSE - Getting Started, Plasma Transport, and Plasma Chemistry. Maybe also one like "Software Development" for software contribution questions/suggestions? Most of my posts on the google group were about general coding issues rather than anything specific to plasma transport or chemistry.

**Reply by cticenhour** [reply]
I like that! Including that suggestion, this is what we have: 

![image](https://user-images.githubusercontent.com/16690058/104047843-8c464f00-519f-11eb-9d62-00c73fd9c989.png)

I can make adjustments to it as we go. Thanks!

**Reply by cticenhour** [reply]
Also, FYI - I think collaborators on the code base can make adjustments to posts, lock them, and reorganize as needed.

**Comment by csdechant** [comment]
I am Corey, a PhD student in Nuclear Engineering at NC State University. I have been working with Zapdos for the past 3-4 years. My current work focus on adding more plasma physics phenomena to study low temperature plasmas more accurately, along with a recent validation/verification study for Zapdos. 

As for categories, I agree that starting off like the MOOSE discussion board helpful. Have a Getting Started and General Q&A, and then dividing the categories into different plasma physics. Currently just having Plasma Transport and Plasma Chemistry seems good, and we can add more distinct categories (like CCP, ICP, microwave discharges) when more capability is added to Zapdos. I also like the idea of a Software Development category, since I think that would help with making Zapdos feel like a more open community.

**Reply by cticenhour** [reply]
Thanks! I'm currently thinking of these as the primary categories, and then we can chop them up, add sub-categories, and whatever else we feel we need to. I'm just excited to have questions and discussions tied closely to the code now!

**Comment by Jae-Nam** [comment]
Hello, I'm Jae, a PhD student in Mechanical Engineering at University of Minnesota. I'm just starting studying plasma-liquid interactions and quite interested in using Zapdos for my modeling research. I ran several very basic simulations using Zapdos and presented at my group meeting. This seems to be wonderful tool. Unfortunately I'm not a developer or skilled programmer, but I'd like to share some ideas and I'd be happy to learn more about Zapdos here by asking some silly questions. 
Thank you!

**Reply by lindsayad** [reply]
Hi @Jae-Nam, thanks for introducing yourself. All ideas and questions are welcome!
